utils/coreutils_test.go

```diff
@@ -1120,35 +1121,35 @@ Date:   Fri Dec 30 19:48:09 2016 +0100
 	GitLastLog = ""
 	if vers, err := GetCGRVersion(); err != nil {
 		t.Error(err)
-	} else if vers != "CGRateS@v0.10.2" {
-		t.Errorf("Expecting: <CGRateS@v0.10.2~dev>, received: <%s>", vers)
+	} else if vers != version {
+		t.Errorf("Expecting: <%s>, received: <%s>", version, vers)
```

Comment:
please rename the variable to something more sane (eVersion or expVers). Valid in all places.